checkmates/data_checks/datacheck_meta/data_check_message_code.py

```diff
@@ -58,6 +58,9 @@ class DataCheckMessageCode(Enum):
     TARGET_LOGNORMAL_DISTRIBUTION = "target_lognormal_distribution"
     """Message code for target data with a lognormal distribution."""
 
+    TARGET_SKEWED_DISTRIBUTION = "target_skewed_distribution"
```

Comment:
Should add data check messages for skews in the features as well

Reply:
changed msg code to reflect overall data skews